spec/spec-v1.md

```diff
@@ -318,10 +326,12 @@ Any integration or tool that is considered a client and implements SCIF must pro
  - **optional** an ability to validate the correctness of a recipe or structure of a SCIF
 
 #### Commands
+
 The minimal set of functions for the controlled must support the following higher level commands. We use the controller name `ctrl` in the examples below, and suggested action names:
 
  * **listing** the UA must, with one command (e.g., "apps" or "list" or "apps.list" are recommended but not enforced) show all the software apps installed in a SCIF. E.g., `ctrl apps` or `ctrl apps <container>`.
  * **run** the UA must provide a means for the user to execute the entrypoint for a chosen app (the `apprun` section from the recipe), e.g., `ctrl run example`. For a container, running the container in context of the app should direct to some central entrypoint that then can forward to the targeted app. If no specific apps are provided, the target should end in a reasonable default. 
+ * **start** akin to "run," the UA must provide a means for the user to execute the start script for a chosen app (the `appstart` section from the recipe), e.g., `ctrl start example`. For a container, running the container in context of the app should direct to some central entrypoint that then can forward to the targeted app. If no specific apps are provided, the target should end in a reasonable default. 
```

Comment:
```suggestion
 * **start** akin to "run," the UA can optionally provide a means for the user to execute the start script for a chosen app (the `appstart` section from the recipe), e.g., `ctrl start example`. Since start is optional, the command can be provided from a subcommand or not provided at all.
 ```
What about this instead?

Reply:
That seems fair to me.

Reply:
ok, added.